pkg/server/grpc_server_test.go

```diff
@@ -522,6 +524,9 @@ func TestAPIWithUriSubject(t *testing.T) {
 		t.Fatalf("config.Read() = %v", err)
 	}
 
+	// Give a few seconds for the oidc servers to start up
+	time.Sleep(5 * time.Second)
```

Comment:
It'd be preferable to poll an endpoint until it returns something, maybe do this in newOIDCIssuer?

Reply:
i don't think we need this actually, the bug you caught above was the real issue!